src/schema/v2/me/collectArtistsMutation.ts

```diff
@@ -0,0 +1,65 @@
+import { GraphQLList, GraphQLString } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { GraphQLNonNull } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { meType } from "./index"
+import { UserInterest, userInterestType } from "../userInterests"
+
+interface Input {
+  artistIDs: string[]
+}
+
+export const collectArtistsMutation = mutationWithClientMutationId<
+  Input,
+  UserInterest[] | null,
+  ResolverContext
+>({
+  name: "CollectArtistsMutation",
+  description: "Collect Multiple Artists",
+  inputFields: {
+    artistIDs: { type: new GraphQLNonNull(new GraphQLList(GraphQLString)) },
```

Comment:
nice typing, i think you can even add one more layer, a non-null list of non-null strings:

``` ts
new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(GraphQLString)))
```